### PR TITLE
Added French translation

### PR DESCRIPTION
**Pull request by Creeper09**
This time hopefully for good

### CODE REVIEW

**Comment by rom1504** [comment]
can you add the link to the french readme in the main readme ?

**Comment by Creeper09** [comment]
Do I do the same for every READMEs ?

**Comment by rom1504** [comment]
Ah yeah indeed

File: docs/README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/mineflayer)
 
-| 🇺🇸 [English](README.md) | 🇷🇺 [Russian](ru/README_RU.md) | 🇪🇸 [Spanish](es/README_ES.md) | 🇹🇷 [Turkish](tr/README_TR.md) |
-|-------------------------|----------------------------|----------------------------|----------------------------|
+| 🇺🇸 [English](README.md) | 🇷🇺 [Russian](ru/README_RU.md) | 🇪🇸 [Spanish](es/README_ES.md) | 🇪🇸 [French](fr/README_FR.md) | 🇹🇷 [Turkish](tr/README_TR.md) |
```
Comment (DrMoraschi) [inline docs/README.md:12]: If I recall correctly that's not the French flag🤣

**Review by DrMoraschi — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (Creeper09) [reply docs/README.md]: 😅

**Comment by Creeper09** [comment]
I believe it is all done and dusted

**Review by rom1504 — APPROVED** [review]

**Comment by Creeper09** [comment]
Sweet!

**Pull request merged**